src/models/category.ts

```diff
@@ -1,7 +1,9 @@
 export interface Category {
+    pk: number,
     name: string,
     description: string,
-    parentName: string,
+    parent: number,
     numQues: number,
```

Comment:
Sorry, I really should've caught this before. But 'numQues' is a really bad name. I also commented this on the API pull request. So you have to change it here too.